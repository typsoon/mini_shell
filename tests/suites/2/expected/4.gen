sed 's/lls/ls -1f/g;s/lcd/cd/g' $SUITE_DIR/$INPUT_DIR/$TEST_NUMBER.in | /bin/sh 2> $SUITE_DIR/$EXPECTED_DIR/$TEST_NUMBER.err | grep -vE '^\.' > $SUITE_DIR/$EXPECTED_DIR/$TEST_NUMBER.out 
