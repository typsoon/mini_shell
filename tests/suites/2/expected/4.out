servers
bin
chdir to servers
fs
pm
init
rs
Makefile
chdir to pm
fs
pm
init
rs
Makefile
done
